parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileReader.java

```diff
@@ -450,6 +450,19 @@ public static final ParquetMetadata readFooter(Configuration configuration, File
    * @throws IOException if an error occurs while reading the file
    */
   public static final ParquetMetadata readFooter(long fileLen, String filePath, SeekableInputStream f, MetadataFilter filter) throws IOException {
+    return readFooter(new ParquetMetadataConverter(), fileLen, filePath, f, filter);
```

Comment:
Since this method has not been released yet, you can change its signature.
How about introducing a ParquetReadConfig class that would have the field useSignedStringMinMax (or a better name)?
That way we don't add an unnecessary dependency on hadoop.Configuration.